assets/js/book.js

```diff
@@ -46,10 +46,9 @@ class Book {
     addNewBook.id = newId;
     const bookInfo = `
       <div id="${newId}">
-        <p>${addNewBook.title}</p>
-        <p>${addNewBook.author}</p>
-        <button id="${newId}" class="btn-remove">Remove</button>
-        <hr>
+        <p>"<span>${addNewBook.title}</span>" by
+        <span class="">${addNewBook.author}</span></p>
+        <button id="${newId}" class="remove-btn">Remove</button>
```

Comment:
- [ ] Kindly observe why your app does not allow users to remove the book. I can add books, when the page is reloaded the books are there, which is good, but when I try to delete books using the remove button, nothing happens, both in the local storage and the page. Kindly ensure that your remove feature works as required. The deleted book should be removed from the local storage and the page simultaneously.